databases/week2/CreatingDatabase.sql

```diff
@@ -0,0 +1,43 @@
+CREATE Table `customer_master` (
+customer_number INT(10) AUTO_INCREMENT PRIMARY KEY,
+first_name VARCHAR(10)NOT NULL,
+middle_name VARCHAR(10)NULL,
+last_name VARCHAR(10)NOT NULL,
+customer_city VARCHAR(10)NOT NULL,
+customer_contact_number INT(10) NOT NULL,
+customer_date_of_birth DATETIME NOT NULL
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+CREATE Table `account_master`(
+    account_number INT(10) AUTO_INCREMENT PRIMARY KEY,
+    customer_number INT(10) NOT NULL,
+    branch_id INT(10)NOT NULL,
+    opening_balance DATETIME NOT NULL,
+    account_type VARCHAR(200) NOT NULL,
+    account_opening_date DATETIME NOT NULL,
+    CONSTRAINT `FK_customer_number` FOREIGN KEY (customer_number)
+    REFERENCES customer_master(customer_number),
+    CONSTRAINT `FK_branch_id` FOREIGN KEY (branch_id)
+    REFERENCES branch_master(branch_id)
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+CREATE Table `transaction_details`(
+    transaction_number INT(10) AUTO_INCREMENT PRIMARY KEY,
+    account_number INT(10) NOT NULL,
+    date_of_transaction DATETIME NOT NULL,
+    transaction_amount INT(10) NOT NULL,
+    CONSTRAINT `FK_account_number` FOREIGN KEY (account_number)
+    REFERENCES account_master(account_number)
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+CREATE Table `branch_master`( 
+    branch_id INT(10) AUTO_INCREMENT PRIMARY KEY,
+    branch_name VARCHAR(200) NOT NULL,
+    branch_city VARCHAR(20) NOT NULL
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+CREATE Table `loan_details`(
+    customer_number INT(10) NOT NULL,
+    branch_id INT(10)NOT NULL,
+    loan_amount INT(100),
+    CONSTRAINT `FK_customer_number2` FOREIGN KEY (customer_number)
+    REFERENCES customer_master(customer_number),
+    CONSTRAINT `FK_branch_id2` FOREIGN KEY (branch_id)
+    REFERENCES branch_master(branch_id)
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
Look great design, but not able to see the ERD(not able to open the PDF file)

Reply:
oh no
![Screen Shot 2021-09-30 at 11 18 57 PM](https://user-images.githubusercontent.com/77116575/135750699-6c90095b-9770-4b14-9830-b81715aff40a.png)

Reply:
I hope you can see this one

Reply:
Nice, thanks!

Reply:
Hi Elia thanks for feedbacks , but do you know how I can add my diagram to my database file in VScode? for making my diagram I used dbdiagram.io .